test/gpu/native/streamPrototype/stream.chpl

```diff
@@ -11,6 +11,8 @@ use GPUDiagnostics;
 //
 use HPCCProblemSize;
 
+config param useForeach = true;
```

Comment:
Again, I'm curious if this had a noticeable impact on performance.

I'm fine including it either way since as you said it's a more apples-to-apples comparison.

Reply:
It does have a noticeable impact, but small. I can't find the PR where I posted that plot first, but it is slide 18 [here](https://chapel-lang.org/CHIUW/2022/KayrakliogluSlides.pdf) Again, small to the extent that not visible in the plot, but not noise. Maybe ignorable with larger kernels?